src/components/HTMLEngineProvider/HTMLRenderers/PreRenderer/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import BasePreRenderer from './BasePreRenderer';
+
+class PreRenderer extends React.Component {
+    constructor(props) {
+        super(props);
+        this.onScroll = this.onScroll.bind(this);
+        this.onPress = this.onPress.bind(this);
+    }
+
+    componentDidMount() {
+        if (!this.scrollRef) {
+            return;
+        }
+        this.scrollNode = this.scrollRef.getScrollableNode();
+        this.scrollNode.addEventListener('mousedown', this.onPress);
+        this.scrollNode.addEventListener('wheel', this.onScroll);
+    }
+
+    componentWillUnmount() {
+        this.scrollNode.removeEventListener('wheel', this.onScroll);
+        this.scrollNode.removeEventListener('mousedown', this.onPress);
+    }
+
+    onPress(e) {
+        const isScrollbar = e.target.classList.contains('r-WebkitOverflowScrolling-150rngu');
```

Comment:
We still have references to classes..... Do you see any other way to fix it? Unfortunately, I don't want classes to be introduced in RN code unless there is no other way. Also, this is a dynamic class name that can change at any time resulting in broken code.

Reply:
> I am not able to scroll the content on IOS Sim. I will test android later.
Drawer Navigator has  options -  swipeEdgeWidth: 400.   this option prevents scrolling in Scrollview of PreRenderer. Can we lower this value of 150 ?

Reply:
No there should be another way...lets find it.

Reply:
> Do you see any other way to fix it? 

we need to stop bubbling on the Scrollbar element but I do not know how to find it

Reply:
For this specific case. Can we use PANresponder to control the touch?